documentation/api/io.strimzi.api.kafka.model.KafkaUserQuotas.adoc

```diff
@@ -0,0 +1,22 @@
+Kafka cluster allows user to enforce certain requests to control usage of the resources by clients.
+We can divide quotas to the two groups:
+
+* network usage quotas, which as are defined as the byte rate threshold for each group of clients sharing a quota
+* hardware utilization quotas, which are defined as the percentage of time a client can utilize on request handler I/O threads and network threads of each broker within a quota window
```

Comment:
Is it really _hardware utilization_? I would maybe just call it _utilization_ or maybe _CPU utilization_. Or is ths term from the KAfka docs?